README.md

```diff
@@ -132,39 +132,39 @@ completely implementing every pass.
-| [Assumptions][]                   | :white_check_mark: | :white_check_mark: | :white_check_mark: |   :green_circle:   |    :x: [235][]     |     :x:     |        :x:         |
-| [Lambdas][]                       | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: |     :x:     | :white_check_mark: |
-| [Multiline disjunctions][]        | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: |     :x:     | :white_check_mark: |
-| [Multiline conjunctions][]        | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: |     :x:     | :white_check_mark: |
-| [Delayed assignment][]            | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: |     :x:     | :white_check_mark: |
-| Invariant checking                |         -          |         -          |         -          |         -          | :white_check_mark: |     :x:     | :white_check_mark: |
-| [Higher-order definitions][]      | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: |    :x: [221][]     |     :x:     |        :x:         |
-| [Runs][]                          | :white_check_mark: | :white_check_mark: |   :green_circle:   | :white_check_mark: | :white_check_mark: |     :x:     | :white_check_mark: |
-| [Temporal operators][]            | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: |     *non-goal*     |     :x:     |        :x:         |
-| [Fairness][]                      | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: |     *non-goal*     |     :x:     |        :x:         |
-| [Unbounded quantifiers][]         | :white_check_mark: | :white_check_mark: |        :x:         |        :x:         |     *non-goal*     |     :x:     |        :x:         |
-| [String literals][], see #118     | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: |     :x:     | :white_check_mark: |
-| ~~uninterpreted types~~, see #118 | :white_check_mark: | :white_check_mark: |        :x:         |        :x:         |        :x:         |     :x:     |        :x:         |
+| Language feature                  | Parser             | Name resolution    | Effects            | Type checker       | Simulator          | To-Apalache        | Tutorials          |
+|:----------------------------------|:------------------:|:------------------:|:------------------:|:------------------:|:------------------:|:------------------:|:------------------:|
+| [Booleans][]                      | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :x:                | :white_check_mark: |
```

Comment:
I believe other rows are also already supported by Apalache conversion. Can you update those?

Reply:
I’ll do that a in a follow up.